app/pages/Accounts/useAccountSync.ts

```diff
@@ -35,9 +36,31 @@ export default function useAccountSync(): string | undefined {
     const dispatch = useDispatch();
     const account = useSelector(chosenAccountSelector);
     const accountInfo = useSelector(chosenAccountInfoSelector);
-    const [controller] = useState(new AbortController());
+    // This controller is used to abort updateTransactions, when the chosen account is changed, or the view is destroyed.
+    const [updateTransactionsController] = useState(new AbortController());
+    // This controller is used to control when the wallet should load a batch of the newest transactions.
+    const [newestTransactionsController] = useState(new AbortController(false));
```

Comment:
Try to do it without having a second controller, as we usually use controllers to make sure we don't have leaks, and not to control when something should start running.

Reply:
I have removed the second controller, and added another field to the original controller.